packages/FormElement/src/components/Content/Content.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ContentContainerStyled from "./Content.styles";
+
+const propTypes = {
+  children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
+  idForLabel: PropTypes.string,
+  refLabel: PropTypes.shape({ current: PropTypes.instanceOf(Object) }),
+  ariaDescribedBy: PropTypes.string,
```

Comment:
Should these props be all required since Content is always inside FormElement

Reply:
![image](https://user-images.githubusercontent.com/4040102/72569794-1a7b4400-3870-11ea-9568-0ff63f406c21.png)
I originally wanted them to be required but run into compiling warnings as they are added to the cloned element and not set by the consumer. Any ideas on how to avoid them?

Reply:
Oh I see, I forget they're empty initially, I don't have ideas 😄

Reply:
Yeah i've been running into that issue on other components after using `React.cloneElement` so would be great if someone had a suggestion for dealing with this!